build_centreon_plugins_64.bat

```diff
@@ -82,6 +82,7 @@ CMD /C %PERL_INSTALL_DIR%\perl\site\bin\pp --lib=centreon-plugins\ ^
 -M Net::FTPSSL ^
 -M Net::NTP ^
 -M Net::SSLeay ^
+-M Net::LDAP ^

```

Comment:
Try to keep in alphabetical order ?
`Safe` a few line below should be re-ordered too :)
Btw, which plugin Net::LDAP is missing for ?
Thx 👍